src/store/addressesSlice.ts

```diff
@@ -105,18 +104,9 @@ export const fetchAddressesDataNextPage = createAsyncThunk(
           : data.balance
         : undefined
 
-      const address = state.addresses.entities[addressHash]
-      const allPagesLoaded = address?.networkData.transactions.allPagesLoaded
-      const latestPage = address?.networkData.transactions.loadedPage ?? 0
-      let nextPage = latestPage
-      let newTransactions = [] as Transaction[]
-
-      if (!allPagesLoaded) {
-        nextPage += 1
-        console.log(`⬇️ Fetching page ${nextPage} of address confirmed transactions: `, addressHash)
-        const { data: transactions } = await client.explorerClient.getAddressTransactions(addressHash, nextPage)
-        newTransactions = transactions
-      }
+      const page = 1
+      console.log(`⬇️ Fetching page ${page} of address confirmed transactions: `, addressHash)
```

Comment:
Should we differentiate the log message from the one from `fetchAddressesDataNextPage`?
We fetch more than just the transactions here.
Also, if `page` is a constant, let's just remove it?

Reply:
Yes, I'll rework it a bit, thanks!

Reply:
Same for this fix :) a0a2b3f